### DISCUSSION TITLE
Add to Site Games List: Mastered Column

### DISCUSSION THREAD

**Discussion by Altomar79** [RetroAchievements/RAWeb#697] [Ideas]
For completionist it would be nice to know while browsing the list with all supported games (per console) which ones he has already mastered or maybe the persent/amount of the achievement he got just by adding one more column.

A nice example can be seen at [Playstation Network Profiles](https://psnprofiles.com/Andritso?pf=ps3) with a trophy thats is brighter when you have 100% for a game.

**Comment by Jamiras** [comment]
Done in #2908.
